sampledata.js

```diff
@@ -0,0 +1,48 @@
+const sampleFeed = [
+  {
+    transactionId: 123,
+    amount: 12.45,
+    timestamp: '2017-102 T 10:15 UTC',
+    note: '🍴 Dinner Plans!',
+    payee: {
+      userId: 1234,
+      username: 'sarah-jacobs',
+      fullName: 'Sarah Jacobs',
+      firstName: 'Sarah',
+      lastName: 'Jacobs'
+    },
+    payer: {
+      userId: 1435,
+      displayName: 'Daniel Rimmings',
```

Comment:
do we need displayName, fullName, firstName and lastName? Lots of redundancies. We discussed the client would be dumb and we'd only send the displayName, right? Are displayName and fullName always the same?

Reply:
@jGlass314 My bad. I thought I fixed this after our morning meeting. I must have missed this one. I will fix this.